ui/apps/platform/src/Containers/PolicyManagement/PolicyManagementPage.tsx

```diff
@@ -12,6 +18,9 @@ function PolicyManagementPage() {
             <Route path={policiesPath}>
                 <PoliciesPage />
             </Route>
+            <Route path={policyCategoriesPath}>
+                <PolicyCategoriesPage />
+            </Route>
```

Comment:
Does policy categories route need to have conditional rendering according to feature flag?

Comment about preceding unchanged line 17 can you verify that it is **not** needed here, because I think the `Redirect` element in Body.tsx is enough.

Reply:
hmm good question. i can check without the flag just in case, but i'm under the impression that it's okay

Reply:
yeah it is, i'll remove it